test/plots/penguin-quantile-unknown.js

```diff
@@ -0,0 +1,14 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function () {
+  const sample = await d3.csv("data/penguins.csv", d3.autoType).then((data) =>
+    data.map((d, i) => ({
+      ...d,
+      body_mass_g: i % 7 === 0 ? NaN : d.body_mass_g
+    }))
+  );
```

Comment:
I think (await d3.csv(…)).map will be a little shorter than await d3.csv(…).then((data) => data.map(…)).